lib/banyan_api/storefront_event.ex

```diff
@@ -28,6 +30,7 @@ defmodule BanyanAPI.StorefrontEvent do
         email: shop.settings["email"],
         shop_myshopify_domain: shop.settings["myshopify_domain"],
         shopify_order: Jason.encode!(shopify_order),
+        recipient_email: recipient_email,
```

Comment:
this would then become something like `recipient_email: Map.get(params, :recipient_email, "")`